### ISSUE TITLE
Getting "bottle :unneeded is deprecated" warnings 

### ISSUE THREAD

**Issue by agrumer** [oncletom/homebrew-brew#5]
I’ve been getting warnings during brew upgrade: 

```
Warning: Calling bottle :unneeded is deprecated! There is no replacement.
Please report this issue to the oncletom/cpdf tap (not Homebrew/brew or Homebrew/core):
  /usr/local/Homebrew/Library/Taps/oncletom/homebrew-cpdf/Formula/cpdf.rb:8
```

**Comment by alexclst** [comment]
I see the same warnings. Homebrew version:

```
Homebrew 3.3.2
Homebrew/homebrew-core (git revision 77fc3850187; last commit 2021-11-01)
Homebrew/homebrew-cask (git revision f228cc2195; last commit 2021-11-02)
```

**Comment by thom4parisot** [comment]
Thanks for raising this to my attention, I also faced the warnings 😅 

Fixed in b3d2509.
